src/jit/lsraarm.cpp

```diff
@@ -112,6 +112,46 @@ void LinearScan::BuildLclHeap(GenTree* tree)
     }
 }
 
+//------------------------------------------------------------------------
+// BuildShiftLongCarry: Set the node info for GT_LSH_HI or GT_RSH_LO.
+//
+// Note: these operands have interfering uses and need the special handling.
```

Comment:
I think it would be more correct to say that they have uses that interfere with the def (uses are always considered to interfere with each other).
Also, please put the note below the arguments as per the function header guidelines.
Otherwise, LGTM

Reply:
Thank you, fixed.